connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -106,13 +106,23 @@ export async function githubUpsertIssueActivity(
       page: resultPage,
     });
     resultPageLogger.info("Fetching GitHub issue comments result page.");
-    const comments = await getIssueCommentsPage(
-      installationId,
-      repoName,
-      login,
-      issueNumber,
-      resultPage
-    );
+    let comments = undefined;
+    try {
+      comments = await getIssueCommentsPage(
+        installationId,
+        repoName,
+        login,
+        issueNumber,
+        resultPage
+      );
+    } catch (e: any) {
+      if (e.status === 404) {
+        // Github may return a 404 on the first page if the issue has no comments
```

Comment:
I hope we won't run into cases where we get transient 404s

Reply:
So do I!